README.md

```diff
@@ -89,6 +89,27 @@ GovukError.configure do |config|
 end
 ```
 
+And you can exclude errors from being reported if they occur during the nightly data sync:
```

Comment:
Might want to add a quick qualifier that this applies to integration and staging.